rllib/core/models/configs.py

```diff
@@ -92,16 +92,36 @@ class MLPHeadConfig(ModelConfig):
         ReLU()
         Linear(8, 2)
 
+    Example:
+
+        Configuration:
+        input_dims = [2]
+        hidden_layer_dims = [10]
+        hidden_layer_activation = "silu"
+        hidden_layer_use_layernorm = True
+        output_dims = [2]
```

Comment:
For these examples can we create a code-block that actually creates these settings? 

```
.. code-block: python
     config = MLPHeadConfig(
           input_dims = [2],
           ...
     )
     model = config.build()

     # Resulting stack in pseudocode:
     # ...
     # Linear(10, 2)
```